gensim/models/atmodel.py

```diff
@@ -107,104 +154,60 @@ def construct_author2doc(doc2author):
-        data may be cumbersome as it requires all the existing data to be
-        re-serialized.
-
-        `serialization_path` must be set to a filepath, if `serialized = True` is
-        used. Use, for example, `serialization_path = /tmp/serialized_model.mm` or use your
-        working directory by setting `serialization_path = serialized_model.mm`. An existing
-        file *cannot* be overwritten; either delete the old file or choose a different
-        name.
-
-        Example:
-
-        >>> model = AuthorTopicModel(corpus, num_topics=100, author2doc=author2doc, id2word=id2word)  # train model
-        >>> model.update(corpus2)  # update the author-topic model with additional documents
-
-        >>> model = AuthorTopicModel(
-        ... corpus, num_topics=50, author2doc=author2doc, id2word=id2word, alpha='auto', eval_every=5)
+        Parameters
+        ----------
+        corpus : iterable of list of (int, float), optional
+            Corpus in BoW format
+        num_topics : int, optional
+            Number of topics to be extracted from the training corpus.
+        id2word : :class:`~gensim.corpora.dictionary.Dictionary`, optional
+            A mapping from word ids (integers) to words (strings).
+        author2doc : dict of (str, list of int), optional
+            A dictionary where keys are the names of authors and values are lists of document IDs that the author
+            contributes to.
+        doc2author : dict of (int, list of str), optional
+            A dictionary where the keys are document IDs and the values are lists of author names.
+        chunksize : int, optional
+            Controls the size of the mini-batches.
+        passes : int, optional
+            Number of times the model makes a pass over the entire training data.
+        iterations : int, optional
+            Maximum number of times the model loops over each document.
+        decay : float, optional
+            Controls how old documents are forgotten.
+        offset : float, optional
+            Controls down-weighting of iterations.
+        alpha : float, optional
+            Hyperparameters for author-topic model.Supports special values of 'asymmetric'
+            and 'auto': the former uses a fixed normalized asymmetric 1.0/topicno prior,
+            the latter learns an asymmetric prior directly from your data.
+        eta : float, optional
+            Hyperparameters for author-topic model.
+        update_every : int, optional
+            Make updates in topic probability for latest mini-batch.
+        eval_every : int, optional
+            Calculate and estimate log perplexity for latest mini-batch.
+        gamma_threshold : float, optional
+            Threshold value of gamma(topic difference between consecutive two topics)
+            until which the iterations continue.
+        serialized : bool, optional
+            Indicates whether the input corpora to the model are simple lists
```

Comment:
What is "input corpora"? Where does that come from?

What does "or saved to the hard-drive" mean? If false, "input corpora" are saved to a hard-drive? Why are we saving input?

The documentation of these two parameters (`serialized` and `serialization_path`) is confusing. Who should use this and why?

This is important because the docstring example at the top actually uses these parameters (that's how I found it).